app/src/test/java/org/simple/clinic/activity/TheActivityControllerTest.kt

```diff
@@ -249,6 +251,29 @@ class TheActivityControllerTest {
     assertThat(controller.initialScreenKey()).isInstanceOf(RegistrationPhoneScreenKey::class.java)
   }
 
+  @Test
+  fun `when user is denied access then access denied screen should show`() {
```

Comment:
Should we also add a corresponding test for hiding the access denied screen?

Reply:
Done.